### PR TITLE
Improvements to the theme

### PR DESCRIPTION
**Pull request by dorianneto**
The improvements are:
- SEO
- Widgets (search, categories and tagcloud)
- Support for new language (PT)
- Added support to Addthis
- Added support to icons on main menu
- Added more share icons on footer
- Possibility to hide comments in post or page

I hope you like :)

### CODE REVIEW

**Comment by klugjo** [comment]
@dorianneto 

Thanks, for you contribution. I really appreciate it :)

I am just having an issue with the widgets you created. This is what I get:

http://pasteboard.co/RgheiWb.png

Can you fix it ?

I suggest you remove the widgets because it would force you to modify the layout too much.

This theme is not meant to support widgets (no sidebar).

See what I mean ?

**Comment by dorianneto** [comment]
I understand :)

I removed widgets like you suggested.

**Comment by klugjo** [comment]
Cool. Thanks. 

Merged :)

File: layout/_partial/head.ejs [hunk 1]
```diff
@@ -23,18 +23,37 @@
...
-        <meta property="og:image" content="<%= page.cover %>" />
+    <% if(page.share_cover) { %>
+        <meta property="og:image" content="<%= config.url %><%= page.share_cover %>" />
     <% } else { %>
-        <meta property="og:image" content="<%= config.cover %>"/>
+        <meta property="og:image" content="<%= config.url %><%= config.cover %>"/>
     <% } %>
 
+    <meta name="twitter:card" content="summary" />
+    <meta name="twitter:site" content="@doriansneto" />
```
Comment (BrunoCartier) [inline layout/_partial/head.ejs:55]: Shouldn't this be configurable, instead of your username everytime?

Comment (klugjo) [inline layout/_partial/head.ejs:55]: Agreed. I will take care of that

Comment (dorianneto) [inline layout/_partial/head.ejs:55]: Sorry for this, I really forgot it remove :/

Comment (klugjo) [inline layout/_partial/head.ejs:55]: Ok this is fixed. Let me know if you see something else.

Thanks agains folks

**Pull request merged**